assets/app/views/create/next-steps.html

```diff
@@ -71,8 +71,19 @@ <h1 ng-if="!pendingTasks(tasks()).length && erroredTasks(tasks()).length">Comple
               <div ng-if="createdBuildConfigWithGitHubTrigger()" class="col-md-6">
                 <h2>Making code changes</h2>
                 <div ng-repeat="trigger in createdBuildConfig.spec.triggers" ng-if="trigger.type == 'GitHub'">
-                  <p>A GitHub <a target="_blank" href="{{'webhooks' | helpLink}}">webhook trigger</a> has been created for the <strong>{{createdBuildConfig.metadata.name}}</strong> build config. You can now set up the webhook in your GitHub repository settings in <a target="_blank" href="{{createdBuildConfig.spec.source.git.uri | githubLink}}/settings/hooks">{{createdBuildConfig.spec.source.git.uri | githubLink}}/settings/hooks</a>, using the following payload URL:</p>
-                  <pre class="code">{{createdBuildConfig.metadata.name | webhookURL : trigger.type : trigger.github.secret : projectName}}</pre>
+                  <p>
+                    A GitHub <a target="_blank" href="{{'webhooks' | helpLink}}">webhook trigger</a> has been created for the <strong>{{createdBuildConfig.metadata.name}}</strong> build config.
+                  </p>
+                  <p ng-if="fromSampleRepo">
+                    Since you probably don't own the <a target="_blank" href="{{createdBuildConfig.spec.source.git.uri | githubLink}}">{{createdBuildConfig.spec.source.git.uri | githubLink}}</a> repository used to create this build config, you first need to fork it to an OpenShift-visible git account that you control. After that, <a href="{{createdBuildConfig | editResourceURL}}">edit the <strong>{{createdBuildConfig.metadata.name}}</strong> build config</a> to point to your forked repository. This will most likely start a new build. 
+                  </p>
+                  <p ng-if="fromSampleRepo">
+                    After that you can finally set up the webhook in the forked GitHub repository settings, using the following payload URL:
```

Comment:
"Then you can configure the webhook in the forked repository's settings, using the following payload url:"